src/clients/HubPoolClient.ts

```diff
@@ -1,3 +1,8 @@
+import { Contract, BigNumber, Event, EventFilter } from "ethers";
```

Comment:
Can you get these from `../utils`?

Reply:
I checked a few other subdirectories in sdk-v2 and they all seem to be importing objects directly from ethers, rather than via utils. I'll leave it as-is for now, but we should discuss further to determine whether we want to adjust that strategy.